pkg/controller/workload/workload_processor.go

```diff
@@ -141,6 +142,16 @@ func (p *Processor) storePodFrontendData(uid uint32, ip []byte) error {
 }
 
 func (p *Processor) removeWorkloadResource(removedResources []string) error {
+	for _, uid := range removedResources {
+		p.WorkloadCache.DeleteWorkload(uid)
+		if err := p.removeWorkloadResourceByUid(uid); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (p *Processor) removeWorkloadResourceByUid(uid string) error {
```

Comment:
suggest renaming to removeWorkloadFromBpfMap to distinguish with remove from local cache

Reply:
In fact, the same interface is called, I split it out and reused it